kernel/os/syscfg.yml

```diff
@@ -39,6 +39,12 @@ syscfg.defs:
     OS_SCHEDULING:
         description: 'Whether OS will be started or not'
         value: 1
+    OS_CTX_SW_STACK_CHECK:
+        description: 'Whether to do stack sanity check during context switch'
+        value: 1
```

Comment:
I'd make this disabled by default since it decreases effective stack size so existing apps may break.